neonSlack.css

```diff
@@ -1,8 +1,7 @@
 @-moz-document regexp("https\:\/\/.*\.slack\.com\/messages.*") {
     body {
-        background: rgba(0, 0, 0, 1.0);
+        background: linear-gradient(rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.1)), url('https://raw.githubusercontent.com/Matthew-Smith/neon-slack/master/slackBG.jpg');
```

Comment:
I think the no glow version should remain background free. The glowing helps create a contrast between the background and the text which doesn't happen here.